schemas/pulse-job.yml

```diff
@@ -204,28 +241,180 @@ properties:
   runMachine:
     $ref: "#/definitions/machine"
 
-  artifacts:
-    type: "array"
+  textLogSummary:
```

Comment:
I have similar concerns regarding `bugSuggestions` and `jobInfo`... but again... let's this working and then perfect it later...

I guess we can always mark the fields as "deprecated" when we have a better format...